isolate.sh

```diff
@@ -552,6 +556,33 @@ function wait_for_condition() {
     fi
 }
 
+function cleanup_deployment() {
+    local name=$1
+    local namespace=$2
+    info "Deleting existing Deployment ${name} in namespace ${namespace}..."
+    ${OC} delete deployment ${name} -n ${namespace} --ignore-not-found
+
+    wait_for_no_pod ${namespace} ${name}
```

Comment:
We might wanna remove this additional wait condition `wait_for_no_pod`, it is usually redundant.
If the deployment is removed, their pods should be gone as well

Reply:
I'm going to remove it